src/components/Pagination.ts

```diff
@@ -25,7 +27,7 @@ const Pagination = () => {
         'carousel__pagination-button': true,
         'carousel__pagination-button--active': isActive(slide),
       },
-      'aria-label': `Navigate to slide ${slide + 1}`,
+      'aria-label': config.labels?.ariaNavigateToSlide + ' ' + (slide + 1).toString(),
```

Comment:
I used some kind of template to allow for simpler translations of `Item X of Y`, we perhaps want something similar here, too.

Reply:
Yes, It's better to have a unified API